src/Card/README.md

```diff
@@ -670,3 +671,53 @@ it is meant to be used as a single horizontal row of Cards, not as a grid. See C
   </Card>
 </CardDeck>
 ```
+
+### Card status
+
+Note that in the example below, the content of `Card` is wrapped inside `Card.Body`. The `d-flex` class is added for the `horizontal` orientation to achieve horizontal variant. The `flex-column` class is added for the main `Card` component.
+
+```jsx live
+() => {
+  const [orientation, setOrientation] = useState('vertical');
+  
+  const handleChangeOrientation = (e) => setOrientation(e.target.value);
+  
+  return (
+    <>
+      <Form.Group>
+        <Form.Label>Choose orientation</Form.Label>
+        <Form.RadioSet
+          name="orientation"
+          onChange={handleChangeOrientation}
+          value={orientation}
+        >
+          <Form.Radio value="vertical">Vertical</Form.Radio>
+          <Form.Radio value="horizontal">Horizontal</Form.Radio>
+        </Form.RadioSet>
+      </Form.Group>
+      <Card orientation={orientation} className="flex-column">
+        <Card.Body className={orientation === 'horizontal' ? 'd-flex' : ''}>
```

Comment:
<img width="966" alt="image" src="https://user-images.githubusercontent.com/2828721/181748813-2082cdb0-b8f1-467a-bf3c-6b0bdaaf5a64.png">

I'm unsure about the "d-flex" class on the `Card.Body` here. I think the intention is that the header title would appear above the section title, like so:

<img width="960" alt="image" src="https://user-images.githubusercontent.com/2828721/181749353-e2fa1bb6-514f-4b16-9d90-b78d07e9aaee.png">

Reply:
Put the `Card.Header` out of the `Card.Body`